tests/test_shipitapi.py

```diff
@@ -0,0 +1,193 @@
+import datetime
+import json
+import mock
+import pytest
+import redo
+import requests
+
+
+# mock redo library, to make test faster
+real_retry = redo.retry
+
+
+def fake(*args, **kwargs):
+	kwargs.pop('sleeptime')
+	kwargs.pop('max_sleeptime')
+	return real_retry(sleeptime=1, max_sleeptime=1, jitter=1, *args, **kwargs)
+
+
+redo.retry = fake
+
+
+def test_simple():
```

Comment:
I guess we can remove this test, I suppose it was for testing purposes, right?

Reply:
I previously thought that this test would help any runtime issue, but I guess you are correct, we can use any test to do it. I'll remove it!